codersdk/workspacedisplaystatus.go

```diff
@@ -0,0 +1,48 @@
+package codersdk
+
+// Maps workspace transition to display status for Running job status
+var runningStatusFromTransition = map[WorkspaceTransition]string{
+	WorkspaceTransitionStart:  "Starting",
+	WorkspaceTransitionStop:   "Stopping",
+	WorkspaceTransitionDelete: "Deleting",
+}
+
+// Maps workspace transition to display status for Succeeded job status
+var succeededStatusFromTransition = map[WorkspaceTransition]string{
+	WorkspaceTransitionStart:  "Started",
+	WorkspaceTransitionStop:   "Stopped",
+	WorkspaceTransitionDelete: "Deleted",
+}
+
+const unknownStatus = "Unknown"
```

Comment:
Should we make these enums? Idc if not, just asking.

Reply:
I was thinking about it tbh. Let's do it. All status strings, right?

Reply:
Discussed via DM, and decided to defer this.